src/otx/algo/common/layers/position_embed.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This file has been moved from the 3D object detection module. Reviewing it is optional.

Reply:
On the releases branch PositionEmbeddingLearned was deleted. However, there is no back merge PR yet, so, let's keep it for now. I will remove it later.